recipes/pypdf2_fields/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "PyPDF2_Fields" %}
+{% set version = "1.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/PyPDF2_Fields-{{ version }}.tar.gz
+  sha256: 017f0efcf9625b1e47d000e1c27442358169e98362a55d090280d201f0aaedc4
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - pypdf2 ==1.26.0
+    - python >=3.6
+
+test:
+  imports:
+    - PyPDF2_Fields
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/GRV96/PyPDF2_Fields
+  summary: >-
+    Library PyPDF2_Fields is a complement to PyPDF2. It helps
+    using a PDF file's fields by facilitating several tasks.
+  description: "See the README in the GitHub repository (home)."
```

Comment:
I think you should either put a meaningful description here or remove that field.

Reply:
I just pushed to my fork GRV96/staged-recipes, but this pull request was not updated.